gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/EncassLinker.java

```diff
@@ -27,6 +33,18 @@ public void link(Encass encass, Bundle bundle, Bundle targetBundle) {
         if (policy == null) {
             throw new LinkerException("Could not find policy for Encapsulated Assertion: " + encass.getName() + ". Policy ID: " + encass.getPolicyId());
         }
+        Element encassPortalIntegrationElement = null;
+        Element encassPortalIntegrationEnabledElement = null;
+        try {
+            encassPortalIntegrationElement = getSingleElement(policy.getPolicyDocument(), API_PORTAL_ENCASS_INTEGRATION);
+        } catch (DocumentParseException ex) {
+            // do nothing
+        }
+        if (encassPortalIntegrationElement != null) {
+            encassPortalIntegrationEnabledElement = getSingleChildElement(encassPortalIntegrationElement, ENABLED, true);
+            policy.getPolicyDocument().getFirstChild().removeChild(encassPortalIntegrationElement);
+        }
+        encass.getProperties().put(PORTAL_TEMPLATE, encassPortalIntegrationElement != null && encassPortalIntegrationEnabledElement == null ? "true" : "false");
```

Comment:
did not understand logic. Why do we have set to it to true when encassPortalIntegrationEnabledElement is null

Reply:
@uppoju this child element is set only when assertion is disabled, so if it is set we should consider it as "false", otherwise "true"

Reply:
OK